### ISSUE TITLE
Harmonization for European Accessibility Act

### ISSUE THREAD

**Issue by cmussi** [w3c/epub-3-wg-charter#42]
I suggest to add to the scope of the Group:

- Harmonization of the accessibility requirements to the requirement of the so called European Accessibility Act (Directive (EU) 2019/882 of the European Parliament and of the Council of 17 April 2019 on the accessibility requirements for products and services) https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2019.151.01.0070.01.ENG&toc=OJ:L:2019:151:TOC

**Comment by iherman** [comment]
@avneeshsingh you should probably sync this with what you intent to provide as a text before I create a PR...

**Comment by brewerj** [comment]
Viewing the current text in the scope section with regard to EPUB Accessibility, I would like to suggest a few clarifications: 

> Improvements of the EPUB Accessibility specification; these may include, but not limited to:
verify that EPUB Accessibility 1.X meets the EU Directive for accessibility requirements for eBooks (European Accessibility Act, Directive 2019/882 of the European Parliament); the WG will conduct a review of the EPUB Accessibility specification for meeting the EU requirements, provide clarifications, and add the missing pieces if required;

I suggest, instead of "meets the EU Directive" to use the following phrase: "aligns with.... to the extent possible" rather than "meets the EU Directive..."

Also , instead of "for meeting the EU requirements, provide clarifications, and add the missing pieces if required," I suggest using: "to ensure that it aligns with EU accessible publishing requirements to the extent these improve accessibility;...."
